### ISSUE TITLE
Any good reason to pin down the dependencies?

### ISSUE THREAD

**Issue by bhodorog** [MindscapeHQ/raygun4py#60]
I notice the dependencies of this library are being aggressively pinned down.
```python
'jsonpickle == 0.9.2',
'blinker == 1.3.0',
'requests == 2.9.1'
```
Now, while this is a very good practice for application-like projects, for libraries (such as raygun4py) this is actually not recommended, unless extremely good reasons exists. 

Just to better explain a use case, in my flask powered project, I'm trying to use the latest flask today with the latest blinker today, but this library will force me to use blinker==1.3.0. 

If there are no any other reasons to do that, can you please unlock the dependencies and let the clients of your library to worry about pinning the versions?

**Comment by fundead** [comment]
You're quite correct there, those dependencies shouldn't be pinned to those specific versions. We'll merge a PR with this change in the near term.

Callum
Raygun Limited

**Comment by bhodorog** [comment]
Thanks, sounds good, let me know if you want me to create the PR.

Cheers, 
Bogdan

**Comment by bhodorog** [comment]
P.S. I've just realized one of the 3 pinned down dependencies is `requests`. Now, `blinker` was somehow easy to work around `requests` being used by almost all projects making http requests is quite difficult. My point is a new release unlocking dependencies would be more welcome sooner rather than later.

Thanks, 
Bogdan

**Comment by fundead** [comment]
Thanks for your patience on this one - this has now been released to PyPI as v3.1.5. If in the event the dependencies still do not work let us know.
